ops/storage-ddls/cassandra-setup.cql

```diff
@@ -79,87 +76,126 @@ CREATE TYPE features (
 );
 
 CREATE TABLE computedtiles (
+    tileid text,
     tilex int,
     tiley int,
```

Comment:
I wonder if it would be clearer to use zoom/row/column instead of x/y/z to keep nomenclature similar to the geotile library that we're using.